src/main/java/dev/galacticraft/mod/misc/cape/CapesLoader.java

```diff
@@ -24,53 +24,38 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import dev.galacticraft.mod.api.cape.CapeListener;
-import dev.galacticraft.mod.api.cape.model.CapeModel;
+import dev.galacticraft.mod.Galacticraft;
+import net.minecraft.util.Util;
 import org.apache.commons.io.IOUtils;
 
 import java.io.IOException;
 import java.net.URL;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Map;
 
 /**
- * @author <a href="https://github.com/TeamGalacticraft">TeamGalacticraft</a>
+ * @author <a href="https://github.com/StellarHorizons">StellarHorizons</a>
```

Comment:
Stellar Horizons?

Reply:
That's what I get for copying and pasting at 11pm

Reply:
Resolved in df82743